### PR TITLE
Add support for nested data objects within templates

### PR DESCRIPTION
**Pull request by eoinmcafee00**
Initial template design didn't support nested data values such as:

`kind: template load: test.jsonnet data: builds: - name: build1 - name: build2
`
This PR adds this support.
Ticket was raised at: https://discourse.drone.io/t/drone-not-parsing-template-data-correctly/9531

### CODE REVIEW

File: plugin/converter/starlark/args.go [hunk 1]
```diff
@@ -15,50 +15,98 @@
...
-
-// TODO(bradrydzewski) add build id, will always be zero value
-// TODO(bradrydzewski) add build number, will always be zero value
-// TODO(bradrydzewski) add build started, will always be zero value
-// TODO(bradrydzewski) add build finished, will always be zero value
-// TODO(bradrydzewski) add build created, will always be zero value
-// TODO(bradrydzewski) add build updated, will always be zero value
-// TODO(bradrydzewski) add build parent
-// TODO(bradrydzewski) add build timestamp
+type GoSlice struct {
```
Comment (marko-gacesa) [inline plugin/converter/starlark/args.go:28]: what is this for?

Comment (eoinmcafee00) [reply plugin/converter/starlark/args.go]: Good catch Marko, i forgot to delete this.

File: plugin/converter/starlark/args.go [hunk 2]
```diff
@@ -15,8 +15,12 @@
 package starlark
 
 import (
+	"fmt"
+
 	"github.com/drone/drone/core"
 
+	"reflect"
```
Comment (marko-gacesa) [inline plugin/converter/starlark/args.go:22]: order the imports, move this line just below the "fmt" import.

Comment (eoinmcafee00) [reply plugin/converter/starlark/args.go]: done.

**Review by marko-gacesa — APPROVED** [review]

**Pull request merged**